vl-datepicker.src.js

```diff
@@ -32,7 +32,18 @@ Promise.all([
  * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-datepicker/issues|Issues}
  * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-datepicker.html|Demo}
  */
+/**
+ * Date change event
+ * @event VlDatepicker#change
+ * @typedef {Object} DateChangeEvent
+ * @property {Array} selectedDates array containing a single date object if no range, or min and max date if ranges
+ * @property {string} dateString the string as displayed in the input field
+ * @property {null|Value} value null if no date selected or a JSON object with selected date(s)
+ *
+ * @see {@link https://flatpickr.js.org/events/|flatpickr events}
```

Comment:
Deze verwijzing naar flatpickr misschien eerder plaatsen bij de code waar flatpickr gebruikt wordt.

Reply:
Wel ik had het expliciet hier gezet omdat het event dat hier verstuurd wordt de properties `selectedDates` en `dateStr` van het flatpickr event propageert. 

Maar het is voor de gebruiker van de component misschien te verwarrend en in feite hoeft deze niet te weten dat onderliggend flatpickr wordt gebruikt.

Ik verplaats het.